recipes/tensorflow/samples/tensorflow/retail/retail/tensorflow/scorer.py

```diff
@@ -0,0 +1,142 @@
+#####################################################################
+# ADOBE CONFIDENTIAL
+# ___________________
+#
+#  Copyright 2019 Adobe
+#  All Rights Reserved.
+#
+# NOTICE:  All information contained herein is, and remains
+# the property of Adobe and its suppliers, if any. The intellectual
+# and technical concepts contained herein are proprietary to Adobe
+# and its suppliers and are protected by all applicable intellectual
+# property laws, including trade secret and copyright laws.
+# Dissemination of this information or reproduction of this material
+# is strictly forbidden unless prior written permission is obtained
+# from Adobe.
+#####################################################################
+
+from ml.runtime.python.Interfaces.AbstractScorer import AbstractScorer
+from data_access_sdk_python.reader import DataSetReader
+from data_access_sdk_python.writer import DataSetWriter
+import tensorflow as tf
+
+import pandas as pd
+fc = tf.feature_column
+tf.set_random_seed(123)
+
+import os
+import json
+
+class Scorer(AbstractScorer):
+    def score(self, config={}):
+        tf.logging.set_verbosity(tf.logging.ERROR)
+        tf_config = json.loads(os.environ['TF_CONFIG'])
+        tf_config = json.loads('{}')
+        os.environ['TF_CONFIG'] = json.dumps(tf_config)
+
+
+        #########################################
+        # Load Data
+        #########################################
+        prodreader = DataSetReader(client_id=config['ML_FRAMEWORK_IMS_USER_CLIENT_ID'],
```

Comment:
catalog_url needs to be set to read prod data, otherwise it'll default to INT catalog server